source/shared/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -124,6 +124,8 @@ Update the Concordium Ledger app on Windows
 
 For the app to work properly with the current version of the Desktop Wallet, make sure that you update to the latest version of the app.
 
+.. Note:: If you're using a different computer than the one you used when you installed the app, you must install Python3, pip, and the Python tools for ledger (ledgerblue) before updating the app.
```

Comment:
```suggestion
.. Note:: If you're using a different computer than the one you used when you installed the app, you must install Python3, pip, and the Python tools for Ledger (ledgerblue) before updating the app.
```

Reply:
Should we maybe make the part "you must install Python3, pip, and the Python tools" a link, that references the related "Install Python3 and pip" section? I guess people might come back to this after they haven't done this for a while if they get a new computer, and by then they might have forgotten what they are.